package.json

```diff
@@ -1,23 +1,28 @@
 {
   "private": true,
   "workspaces": [
-    "packages/*"
+    "packages/**"
```

Comment:
Does `pnpm` use the `workspaces` key here?

Reply:
No it requires its own file for it

Reply:
something here tho: https://github.com/pnpm/pnpm/issues/2255#issuecomment-576866891 investigating now

Reply:
No it does not. You need a separate yaml file. You can remove the key in package.json too

https://pnpm.io/pnpm-workspace_yaml

Reply:
Gotcha. 

My understanding of the issue i linked is that they do *not* support the property, but they are open to a feature contribution that would autogenerate the workspace file based on that property upon first execution of `pnpm install`.

Reply:
Just add the file for now